unit-tests/live/frames/test-pipeline-start-stop.py

```diff
@@ -0,0 +1,41 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+
+import pyrealsense2 as rs
+from rspy.stopwatch import Stopwatch
+from rspy import test, log
+import time
+import platform
+
+
+# Start depth + color streams and measure the time from stream opened until first frame arrived using pipeline API.
```

Comment:
You also run IMU, better to write `Start all streams...`

Reply:
Changed